articles/laravel-beginners-guide-blogpost/index.md

```diff
@@ -493,7 +493,12 @@ This is the response you will see in your browser:
 #### Working with other methods
 So far we have been working with `get` routes only. The `create()` and `edit()` methods are supposed to show the create and edit forms respectfully.
```

Comment:
@molly80 

Wow you article is quite detailed and informative. I have definitely learned a lot. However, I have a few pointers.  I noted that you included soo many routes in the web.php file. Many developer wish to develop their apps faster. Could you show the user how to use a resource instead. All the routes can be summarized by something like this.
Route::resource('blog', BlogPostController::class);
It can be generated by
php artisan make:controller BlogController --resource

Similarly, the model and migration can be created in one instance by 
php artisan make:model Blog --migration

I believe this changes will make the article more informative. Otherwise, nice work.